### PR TITLE
Remove billing & upgrade references when STRIPE_API_KEY is missing

### PR DESCRIPTION
**Pull request by simonbukin**
This PR takes a stab at removing upgrade and billing references when STRIPE_API_KEY is not set in the `.env` file. #151 

I'm unsure if there are other places that reference billing and upgrade, but I would be happy to update those as well. Just spun up this repo this morning, so I'm unsure if there are other places these are mentioned.

### CODE REVIEW

**Comment by tmyracle** [comment]
There's also an email that's sent on a scheduled job for trial upgrade reminders in email.processor.ts, we can noop that in main.ts in the workers folder if no api key.

**Comment by simonbukin** [comment]
@tmyracle done!

File: apps/workers/src/main.ts [hunk 1]
```diff
@@ -154,11 +154,13 @@ syncInstitutionQueue.add(
  */
 sendEmailQueue.process('send-email', async (job) => await emailProcessor.send(job.data))
 
-sendEmailQueue.add(
-    'send-email',
-    { type: 'trial-reminders' },
-    { repeat: { cron: '0 */12 * * *' } } // Run every 12 hours
-)
+if (env.STRIPE_API_KEY) {
```
Comment (tmyracle) [inline apps/workers/src/main.ts:157]: lets also make sure there's not a default value in the workers .env file that would cause a false positive

Comment (tmyracle) [reply apps/workers/src/main.ts:157]: 🤦  yeah you're right, I was thinking of the NX_STRIPE_SECRET_KEY which I'm not sure if even used anywhere 🤔

Comment (simonbukin) [reply apps/workers/src/main.ts:157]: Yeah, I saw that too... We're probably going to make some updates to the Zod schema once we figure out what each env variable is doing

Comment (tmyracle) [reply apps/workers/src/main.ts:157]: For sure, will leave that to another PR. LGTM!

**Comment by Shpigford** [comment]
Thanks @simonbukin!

**Comment by cmosguy** [comment]
@simonbukin and @Shpigford there is still a strip container in the `docker-compose.yml` .  Should this be removed as well?

**Comment by Shpigford** [comment]
> @simonbukin and @Shpigford there is still a strip container in the `docker-compose.yml` . Should this be removed as well?

No, as we'll still be making use of Stripe for our hosted solution. This PR is just about hiding UI elements.

**Comment by cmosguy** [comment]
Ah understood - ok thank you!

**Pull request merged**